.github/workflows/docs.yml

```diff
@@ -3,15 +3,18 @@ name: docs
 on:
   create:
     tags:
-    # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+      # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
       - b[0-9][0-9][0-9]
       - publish-docs*
   push:
     branches:
       - main
+  pull_request: # Temporary just for test
+    branches:
+      - main
```

Comment:
I added this for the sake of testing in my PR, but if we keep this we wouldn't need to add a test in `.github/workflows/main.yml`